data_submission/Data_Deidentification.md

```diff
@@ -0,0 +1,26 @@
+---
+order: 998
+---
+
+# Data De-identification
+
+!!! **HTAN Centers are responsible for data deidentification.**
+!!!
+
+As outlined in the HTAN [DMSA](https://docs.google.com/document/d/1RPFm9MBJv8DjZmYZyIv0jbjtNJ8fnwGjYDjlK4lL4nc/edit#heading=h.gjdgxs), data submitted to the Data Coordinating Center (DCC) must be fully de-identified.
+
+By signing the HTAN DMSA, HTAN members’ institutional signing officials and PIs accept responsibility for the de-identification of data prior to transfer to the DCC and confirm that: 
+
+- all data disclosed to the DCC (Synapse) are fully de-identified in accordance with HIPAA; 
+
+- all data were collected in accordance with protocols approved by an IRB or its equivalent; 
+
+- all data are consistent with applicable U.S. laws, regulations, and its institutional policies; and 
+
+- an IRB/Privacy Board or equivalent body has assured that submission and subsequent sharing of data are consistent with the Informed Consent of the Data Subject(s) from whom the data were obtained. In addition, the data are protected by an NIH Certificate of Confidentiality.
+
+New HTAN Centers should develop and submit a De-identification Plan using the [HTAN Atlas De-Identification Plan Template](https://docs.google.com/document/d/1jFXYVMhLEGVcMNBh3L-U1rr4B-KLG7gK-ETc1SzMHhs/edit?usp=sharing).
```

Comment:
this was a recommendation from Ashley, I believe.  Would be easiest to let it stay so we don't forget about it later.  I think it should be ok since it refers to "New HTAN Centers" so shouldn't be confusing to anyone?

Reply:
OK. Yes let's leave for now. We will want to revisit that template for sure at the start of renewal but OK to share for now (and confirmed it only has public view access)